src/pages/CreateApplicationPage.jsx

```diff
@@ -920,9 +928,9 @@ function CreateApplicationPage(props) {
                                 </div>
                             </div>
                         </div>
-                        <div className="row justify-content-center font-barlow gx-0">
-                            <div className="col col-md-10 d-flex flex-column h-100 px-4">
-                                <div className="row justify-content-center justify-content-md-start gx-2 gy-4 mb-4">
+                        <div className="row justify-content-center font-barlow gx-0 p-4">
+                            <div className="col col-md-10 d-flex flex-column h-100">
+                                <div className="row justify-content-center justify-content-md-start gx-2 gy-4">
                                     <div className="col-3 col-md-2">
```

Comment:
```suggestion
                                    <div className="col-3 col-xl-2">
```
O mesmo na linha 954!

Reply:
- Corrigido com revisão geral da página;